dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -694,15 +694,13 @@ components:
             - SWL
             - NFL
             - service
-            - cwl
```

Comment:
Ditto @garyluu from @ByteMap 's branch
Must have forgot to regenerate in earlier PR?

Reply:
@denis-yuen ditto